Dockerfile.aarch64

```diff
@@ -1,9 +1,11 @@
 # Original credit: https://github.com/jpetazzo/dockvpn
 
 # Smallest base image
-FROM aarch64/alpine:3.5
+FROM docker.io/project31/aarch64-alpine-qemu:3.5-7
 
-MAINTAINER Kyle Manna <kyle@kylemanna.com>
+RUN [ "cross-build-start" ]
+
+MAINTAINER Kurt Stam <kurt.stam@redhat.com>
```

Comment:
This is fine, but can you use the new `LABEL` approach as `MAINTAINER` is apparently deprecated.  It's also possible to specify two names, you should specify both mine and yours assuming you accept the responsibility and agree with my general approach. :)

See https://docs.docker.com/engine/reference/builder/#maintainer-deprecated